db/pre-re/skill_db.conf

```diff
@@ -24040,16 +24029,16 @@ skill_db: (
 	}
 	AttackType: "Weapon"
 	SkillData1: {
-		Lv1: 3500
-		Lv2: 4000
-		Lv3: 4500
-		Lv4: 5000
-		Lv5: 5500
-		Lv6: 6000
-		Lv7: 6500
-		Lv8: 7000
-		Lv9: 7500
-		Lv10: 8000
+		Lv1: 5500
+		Lv2: 6000
+		Lv3: 6500
+		Lv4: 7000
+		Lv5: 7500
```

Comment:
Because max level of the skill Is 5. If I recall correctly, I once did a pr removing those for all skulls (and there were a lot) but it was decided to let it be since it was not guaranteed everything will work as intended afterwards. Not sure if it was related to the skill db format conversion or something else, maybe @MishimaHaruna remembers